docs/create_admin_account.rst

```diff
@@ -1,30 +1,30 @@
-Create an admin account on the Journalist Interface
+Create an admin account on the *Journalist Interface*
 ===================================================
```

Comment:
Linting is failing due to the title not being long enough: 

```
/src/docs/create_admin_account.rst:2: (WARNING/2) Title underline too short.
```

You can run the linting locally via `make docs-lint` following [the documentation here](https://docs.securedrop.org/en/latest/development/documentation_guidelines.html#testing-documentation-changes).